library/std/src/sys/unix/ext/io.rs

```diff
@@ -58,6 +88,19 @@ pub trait IntoRawFd {
     /// This function **transfers ownership** of the underlying file descriptor
     /// to the caller. Callers are then the unique owners of the file descriptor
     /// and must close the descriptor once it's no longer needed.
+    ///
+    /// # Example
+    ///
+    /// ```no_run
+    /// use std::fs::File;
+    /// use std::os::unix::io::{IntoRawFd, RawFd};
+    ///
+    /// fn main() -> std::io::Result<()> {
+    ///     let f = File::open("foo.txt")?;
+    ///     let raw_fd: RawFd = f.into_raw_fd();
+    ///     Ok(())
+    /// }
```

Comment:
```suggestion
    /// use std::fs::File;
    /// # use std::io;
    /// use std::os::unix::io::{IntoRawFd, RawFd};
    ///
    /// let f = File::open("foo.txt")?;
    /// let raw_fd: RawFd = f.into_raw_fd();
    /// # Ok::<(), io::Error>(())
```

Based on previous suggestion, output:
```rust
use std::fs::File;
use std::os::unix::io::{IntoRawFd, RawFd};

let f = File::open("foo.txt")?;
let raw_fd: RawFd = f.into_raw_fd();
```
Taken from https://doc.rust-lang.org/rustdoc/documentation-tests.html#using--in-doc-tests

Reply:
@GuillaumeGomez ping just in case you missed this and only saw the other message

Reply:
Don't worry, I saw it. Just didn't update the code yet. ;)